### DISCUSSION TITLE
Organisation

### DISCUSSION THREAD

**Discussion by hrenaud** [cnumr/best-practices-wordpress#79] [Q&A] [answered]
Hello,

J'avoue que je suis un peu perdu dans l'organisation, on a des discussions, des issues et un projet.

On suit quoi, où et comment ?

@florinesueur @dNicolle pouvez-vous m'éclairer et peut-être aussi éclairer les autres

**Comment by florinesueur** [comment]
Coucou,

Du coup, j'ai fait une petite vidéo explication et tuto : https://youtu.be/BPOWY_QCc2g
J'espère que c'est plus clair, si ça ne l'est pas n"hésitez pas.

Aussi toute personne qui est débordée avec ces BP, n'hésitez pas à le signaler au plus vite pour une réattribution ou v2.
Pour toutes issues non attribuées, vous êtes libre dans prendre la charge.

**Reply by hrenaud** [reply]
OK, très bien cette vidéo 👌

Si je comprends bien, le process de vie d'un sujet, c'est ça ? J'ai mis des demandes de petites précisions avec

1. Création d'une Discussion, échanges puis on flag à `Answered`. **Qui ferme ?**
2. Création des issues (qui apparaissent dans la vue projet). **1 issue par sujet avec des sous taches ou plusieurs issues avec tache unique ? Qui crée les issues ?**
3. Création d'une branche `feat/***` sur laquelle on travaille.
4. Au fil de l'avancement, on coche les sujets réglés dans l'issue unique ou l'on coche la tâche unique (suivant la réponse du point 2)
5. On fait une MR de sa branch vers `develop`, on l'affecte à un reviewer. **Comment on détermine qui est reviewer ?**
6. Échanges, commentaires, etc. **Comment on sait qu'un sujet est DONE ?**
7. MR validée par le reviewer
8. Je merge dans `main` et je fais le merge de `main` vers `site-gatsby-generator` ce qui déclenche le build et la publication.

Le process étant le même pour les `bp` et `site` je suppose

**Comment by florinesueur** [comment]
Le process est à discussion bien entendu.

1. L'équipe de pilotage du projet
2. 1 issue par sujet avec des sous taches. La personne qui `answered` la discussion en fonction des accords de la discussion. L'issue est assignée soit à la personnage qui a ouvert la discussion soit à la personne en charge de cette partie.
3. Oui, c'est plus propre et évite les conflits
4. C'est à la personne en charge de l'issue de gérer ses tâches (ici j'ai mis des tâches par defaut récurant pour chaque BP). elle doit aussi crée sa branche et son pull request (si ce n'est pas possible, elle indique ces modifs dans l'issue et une personne du comité de pilotage fera le process)
5. L'équipe de pilotage du projet (en free on ne peut mettre qu'un seul reviewer)
6. Quand c'est tranché, soit tout le monde est d'accord soit le comité de pilotage tranche (il faut avancer)

Même process pour `site` en effet.

**Reply by hrenaud** [reply]
👌🏻 c'est très clair

**Reply by hrenaud** [reply]
On pose ça dans le marbre quelque part ?

**Reply by florinesueur** [reply]
Oui, je suis pour, si tu le monde est ok. La vidéo va dans le marbre ou pas ? Si oui, il faudra peut-être la refaire avec un script :)

**Reply by hrenaud** [reply]
Je valide la vidéo dans le marbre, il manque juste l'appel à s'abonner à la chaine et à appuyer sur le pouce 👍🏻 voire, laisser un commentaire.

Je propose de l'ajouter au même readme que celui que j'ai mis à jour ce matin où j'ai ajouté **Comment créer ou mettre à jour des fiches sur le repository GIT**

**Comment by florinesueur** [comment, marked as answer]
@Yanngautreau @yannkozon , en attendant la finalisation de l'écriture du fichier CONTRIBUTIING.md
Le prosess est expliqué à peu près en vidéo ici : https://youtu.be/BPOWY_QCc2g

La version écrite est https://github.com/cnumr/fiches-wordpress-greenit/pull/99

Vous avez dans https://github.com/cnumr/fiches-wordpress-greenit/issues, les tâches qui vous sont assignées en fonction des discussions qu'il y a eu et les créations de bonnes pratiques que vous avez proposées. Si vous n'avez pas le temps avant le 31/10 merci de l'indiquer.

Vous avez une relecture de vos BP sur https://github.com/cnumr/fiches-wordpress-greenit/pulls que vous devez modifier ou valider, si c'est ok pour vous en commentaire. Vous êtes attribué en tant que review quand ça concerne une BP dont vous êtes l'auteur.

Si vous avez besoin d'une visio, n'hésitez pas à me faire signe ou n'hésitez pas à mettre un pouce en l'air si c'est ok pour vous.
